src/omni-slider.js

```diff
@@ -202,6 +204,21 @@ Slider.prototype.init = function(options) {
   }
 };
 
+Slider.prototype._standardizeUSD_ = function(number) {
+
+  var numberinString = number.toString();
```

Comment:
@chrisrng , I will grab what #3 has `eslintConfig` attribute in `package.json` and give `es2015` a shot. The particular compilation error I was encountering before #3 when using `import` in JavaScript, `npm start` watching would give me `Illegal import...`.

But maybe you can take this PR as it is in `es5` and I will do a follow-up refactoring in the next PR?  Do want this stable feature sitting too cold.